atmat/attrack/linepass.m

```diff
@@ -90,7 +90,7 @@
 
 newlattice = double(~keeplattice);
 
-[prefunc,postfunc]=parseargs({function_handle.empty,function_handle.empty},...
+[prefunc,postfunc]=parseargs({cell(),cell()},...

```

Comment:
Works, but isn't there any way to create empty arrays on any type in Octave?

Reply:
There's no `function_handle` class in Octave. In any case, anything empty will work: https://github.com/atcollab/at/blob/master/atmat/attrack/atpass.c#L403-L404